src/2023/index.html

```diff
@@ -63,68 +63,84 @@ <h2 class="highlight eyebrow">The first Swiss Ruby conference</h2>
           <h2>Speakers</h2>
           <div class="grid speakers center">
             <article class="stack speaker-preview">
-              <figure>
-                <img class="speaker-picture" width="200" height="200" alt="Vladimir Dementyev's speaker picture" src="images/speakers/vladimir-dementyev.jpeg">
-              </figure>
-              <div>
-                Vladimir Dementyev
-              </div>
+              <a href="talks#vladimir-dementyev-profiling-ruby-tests">
+                <figure>
+                  <img class="speaker-picture" width="200" height="200" alt="Vladimir Dementyev's speaker picture" src="images/speakers/vladimir-dementyev.jpeg">
```

Comment:
nitpick: wouldn't it be easier to define `width` and `height` in the `speaker-picture` class?

Reply:
The attributes are [sizing hints](https://web.dev/learn/design/responsive-images/#sizing-hints). That way the browser keeps the aspect ratio and enough space for the images before they load. Otherwise there's a jump on the page.